File: cpp/7.cpp
#include <iostream>
#include <math.h>
using namespace std;

bool ft_isPrime(int n)
{
	int i;

	for (i = 2; i < sqrt(n) + 1; i++)
	{
		if (n % i == 0)
			return (false);
	}

	return (true);
}

int main()
{
	int ctr, i;

	ctr = 0;
	i = 1;

	while(true)
	{
		i++;
		if (ft_isPrime(i))
		{
			ctr++;

			if (ctr == 10001)
			{
				cout << i << endl;
				break;
			}
		}
	}

	return (0);
}

File: Python3/7/7.py
#!/usr/bin/python3

def ft_isPrime(n):
	for i in range(2,int(n**0.5) + 1):
		if n % i == 0:
			return (False)
	return (True)

ctr = 0
i = 1
while(True):
	i += 1
	if ft_isPrime(i):
		ctr += 1
		if ctr == 10001:
			print(i)
			break
	